packages/neuron-wallet/tests/block-sync-renderer/indexer-cache-service.intg.test.ts

```diff
@@ -10,6 +10,17 @@ import RpcService from '../../src/services/rpc-service'
 const stubbedGetTransactionFn = jest.fn()
 const stubbedGetHeaderFn = jest.fn()
 const stubbedGetTransactionsByLockScriptFn = jest.fn()
+const stubbedCollectFn = jest.fn(() => {
```

Comment:
I think this is redundant. It has mock at https://github.com/nervosnetwork/neuron/pull/2684/files#diff-200ae59642cd5bd8413700f7b04c2ca44c1e965c58cc03b7a11cd8765b3740e9R147-R161